.eslintrc.js

```diff
@@ -175,6 +175,7 @@ module.exports = {
     "eslint:recommended",
     "plugin:jsdoc/recommended",
     "plugin:prettier/recommended",
+    "plugin:react-hooks/recommended",
```

Comment:
Instead of this, can you instead do:

```js
{
  "plugins": [
    // ...
    "react-hooks"
  ],
  "rules": {
    // ...
    "react-hooks/rules-of-hooks": "warn",
    "react-hooks/exhaustive-deps": "warn"
  }
}
```

That way we'll get the warnings to keep us in check, and when we've eventually fixed all those warnings we can switch it back to this.